ditto/cli.py

```diff
@@ -54,6 +54,9 @@ def cli(ctx, verbose):
 @click.option("--writers", is_flag=True, help="List all available writers")
 @click.pass_context
 def list(ctx, readers, writers):
+    if not readers and not writers:
```

Comment:
Previously, if you did not provide --readers or --writers there would be no output.  Now, it will print all.